### ISSUE TITLE
Gulp error

### ISSUE THREAD

**Issue by Zubrik** [mozilla/nunjucks#714]
I had resolved this problem some time ago: https://github.com/mozilla/nunjucks/issues/676

Now when I run my gulp task, I get this error:

```
Error: (unknown path)
  Error: Cannot use "in" operator to search for "foo" in unexpected types.
```

So it seems I can't do this anymore:

`{% if 'foo' in activePage %}`

Yesterday I updated node.js to v4.4.1, I don't know if that could have something to do with the problem.

Thanks.

**Comment by carljm** [comment]
What type of object is `activePage`? Can you give a full example for reproduction?

Closing pending further info from OP.

**Comment by Zubrik** [comment]
Hi,

`activePage` is a variable which contains a string ("foo bar"). I want to check if "foo" exists in that string, so I do
```
{% set activePage = "foo bar" %}
{% if 'foo' in activePage %}
  <p>foo exists</p>
{% endif %}
```
which returns the error:

```
gulp-notify: [Error running Nunjucks] Error: (unknown path)
  Error: Cannot use "in" operator to search for "foo" in unexpected types.
```

**Comment by ricordisamoa** [comment]
The source is clear: https://github.com/mozilla/nunjucks/blob/3817607b391a290f1313925d84ad2d2da2ca5510/src/lib.js#L288
It must be an array or an object.

**Comment by carljm** [comment]
The source is clear, but it's wrong :/ The `in` operator should work on strings, and it does in Jinja2. And it used to in nunjucks as well, until I merged what was supposed to be just a simple PR to provide better error messages, but I totally missed the fact that that PR should still have allowed support for strings. That's my fault, I'll get it fixed. Sorry about that @Zubrik.

**Comment by carljm** [comment]
Just released v2.4.2 with the fix for this.
